src/DataDog/AuditBundle/EventSubscriber/AuditSubscriber.php

```diff
@@ -70,9 +70,13 @@ public function getUnauditedEntities()
     {
         return array_keys($this->unauditedEntities);
     }
-
+    
     private function isEntityUnaudited($entity)
     {
+        if ($entity instanceof Association || $entity instanceof AuditLog) {
```

Comment:
I'll check and come back later, seams weird that these can be persisted separately as entities. all audit related entries goes directly through SQL statements and should not be managed manually. guess I need to add exception in constructor. ORM initializes it through reflection.